mrmustard/math/lattice/strategies/vanilla.py

```diff
@@ -68,7 +68,7 @@ def vanilla_jacobian(
     dGdc = G / c
 
     # initialize path iterator
-    path = paths.ndindex_path(G.shape)
```

Comment:
thanks, this function wasn't in use so there weren't tests to catch this

Reply:
I noticed because I used it for the parallel strategy to calculate marginals. (I needed the jacobian of the first hypercubes of the parallel simulations)